src/components/UserProfile/ProfileTabs.tsx

```diff
@@ -56,7 +56,6 @@ const ProfileTabs = ({
 		posts: 0,
 		votes: 0
 	});
-	const { id: userId } = useUserDetailsSelector();
```

Comment:
userId has been removed, but it's still being used in line 126, it'll throw an error